cypress/pages/saucedemo/cart.js

```diff
@@ -0,0 +1,42 @@
+class CartPage {
+    elements = {
+        checkoutButton: () => cy.get("[data-test=checkout]"),
+        continueShoppingButton: () => cy.get('[data-test="continue-shopping"]'),
+        cartList: () => cy.get('[data-test="cart-list"]'),
+    }
+
+    getListItem() {
+        return this.elements.cartList().find('.cart_item_label>a>[data-test="inventory-item-name"]')
```

Comment:
Por algun motivo es tan específico este locator? no nos alcanza solo con el data-test?
```suggestion
        return this.elements.cartList().find('[data-test="inventory-item-name"]')
```